.github/workflows/ci.yml

```diff
@@ -647,19 +607,29 @@ jobs:
         uses: "actions/checkout@v4"
         with:
           submodules: true
+      - name: Install NodeJS
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20
+          cache: 'npm'
+          cache-dependency-path: docs/package-lock.json
       - name: Set up Python
         uses: actions/setup-python@v5
         with:
           python-version: "3.12"
-      - name: "Setup Python environment"
-        run: |
-          pipx install -f poetry==${{ needs.prepare-environment.outputs.POETRY_VERSION }}
-          poetry config virtualenvs.create true --local
-          poetry env use 3.12
-      - name: "Install dependencies"
-        run: "poetry install --no-interaction --no-ansi"
-      - name: "Setup environment"
+      - name: Install Poetry
+        uses: snok/install-poetry@v1
```

Comment:
Is using this github action required? Can't we stay consistent with the other method used within this CI pipeline? (I know this one comes from another repo)

Reply:
I've removed it from here, thanks for fixing @fatih-acar !